wasm/src/debug.rs

```diff
@@ -0,0 +1,46 @@
+use super::*;
+
+/// Print `obj` in debug representation to the stdout
+pub fn dbg<T: Debug + ?Sized>(obj: &T) {
```

Comment:
```suggestion
fn dbg<T: Debug + ?Sized>(obj: &T) {
```

can't it be private?

Reply:
The idea is also to use `dbg()` from user code